### PR TITLE
ENCD-5433-refactor-not-bdd-not-indexing-tests

### PR DESCRIPTION
**Pull request by hitz**

### CODE REVIEW

File: src/encoded/tests/features/conftest.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def app(app_settings):
 
 @pytest.mark.fixture_cost(500)
 @pytest.yield_fixture(scope='session')
-def workbook(request, app):
+def index_workbook(request, app):
```
Comment (keenangraham) [inline src/encoded/tests/features/conftest.py:32]: How does renaming help?

File: src/encoded/tests/test_search_views.py [hunk 2]
```diff
@@ -1,9 +1,9 @@
 import pytest
 
-from encoded.tests.features.conftest import app_settings, app, workbook
+from encoded.tests.features.conftest import app_settings, app, index_workbook
 
 
-def test_search_views_search_view_with_filters(workbook, testapp):
+def test_search_views_search_view_with_filters(index_workbook, testapp):
```
Comment (keenangraham) [inline src/encoded/tests/test_search_views.py:7]: > -m “indexing” - all tests now pytest.mark.indexing  - includes test_indexing, test_batchdownload, and test_search_views.

Were these supposed to be marked as indexing? Looks like they are still running in `non bdd non indexing tests`.

**Review by keenangraham — CHANGES_REQUESTED** [review]

File: src/encoded/tests/features/conftest.py (hunk 1 above)
Comment (hitz) [reply src/encoded/tests/features/conftest.py:32]: prevents confusion between the workbook fixture that doesn't index in ES and the one that does

File: src/encoded/tests/test_search_views.py (hunk 2 above)
Comment (hitz) [reply src/encoded/tests/test_search_views.py:7]: it looks like maybe the search_views tests didn't get pytest.marked as indexing?

Comment (hitz) [reply src/encoded/tests/test_search_views.py:7]: fixed

**Review by keenangraham — APPROVED** [review]
Looks like `test_search_views.py` running in `non bdd indexing` now.

**Pull request merged**